src/wp-includes/wp-db.php

```diff
@@ -1495,6 +1502,15 @@ public function prepare( $query, ...$args ) {
 		return $this->add_placeholder_escape( $query );
 	}
 
+	/**
+	 * Similar to $this->prepare(), but returns a WP_DB_Partial_Query object instead of a string.
+	 * This method can be used as a query builder to join prepared query from various places.
+	 */
```

Comment:
```suggestion
	/**
	 * Similar to $this->prepare(), but returns a WP_DB_Partial_Query object instead of a string.
	 * This method can be used as a query builder to join prepared query from various places.
	 *
	 * @return WP_DB_Partial_Query The prepared partial query.
	 */
```

A `@return` annotation should be provided for methods that return something.